src/content/_data/changelog.yml

```diff
@@ -1,4 +1,43 @@
 ---
```

Comment:
@molant, @qzhou1607 Doesn't this file need to be committed?

Reply:
@alrra Nope, its content is updated in each build by the `updateChanglog` method in `updater.js`.

Reply:
Ok. :(